fonts.css

```diff
@@ -0,0 +1,38 @@
+@font-face {
+    font-family: 'PT Serif';
+    src: url('../font/PTSerif-Regular.eot');
+    src: url('../font/PTSerif-Regular.eot?#iefix') format('embedded-opentype'),
+        url('../font/PTSerif-Regular.woff2') format('woff2'),
+        url('../font/PTSerif-Regular.woff') format('woff'),
+        url('../font/PTSerif-Regular.ttf') format('truetype'),
+        url('../font/PTSerif-Regular.svg#PTSerif-Regular') format('svg');
+    font-weight: normal;
+    font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5764748)